File: 백준/Silver/11536. 줄 세우기/줄 세우기.py
names = []
for _ in range(N := int(input())):
    names.append(input())

names_sort = sorted(names)
names_sort_reverse = sorted(names, reverse=True)

if names == names_sort:
    print("INCREASING")
elif names == names_sort_reverse:
    print("DECREASING")
else:
    print("NEITHER")

File: 백준/Silver/11536. 줄 세우기/줄 세우기.js
const [T, ...members] = require("fs")
  .readFileSync(0, "utf-8")
  .trim()
  .split("\n")

const isAscending = (members) => {
  for (let i = 0; i < members.length - 1; i++) {
    if (members[i] > members[i + 1]) return false
  }
  return true
}

const isDescending = (members) => {
  for (let i = 0; i < members.length - 1; i++) {
    if (members[i] < members[i + 1]) return false
  }
  return true
}


const result = isAscending(members)
  ? "INCREASING"
  : isDescending(members)
  ? "DECREASING"
  : "NEITHER"

console.log(result)